Libplanet/Blockchain/BlockChain.cs

```diff
@@ -410,7 +410,9 @@ bool render
         internal void ValidateNonce(Block<T> block)
         {
             var nonces = new Dictionary<Address, long>();
-            foreach (Transaction<T> tx in block.Transactions)
+            IEnumerable<Transaction<T>> transactions =
```

Comment:
it seems to be already sorted by nonce in [`.MineBlock()`](https://github.com/planetarium/libplanet/commit/fbca5c679061a74af55db1994037b10a8da581d7#diff-f9c16aaa8d3a312223ca86876f791202)

Reply:
You're right. This was not the cause of the bug.